src/main/java/net/neoforged/neoforge/event/entity/item/ItemExpireEvent.java

```diff
@@ -6,32 +6,33 @@
 package net.neoforged.neoforge.event.entity.item;
 
 import net.minecraft.world.entity.item.ItemEntity;
-import net.neoforged.bus.api.ICancellableEvent;
 
 /**
  * Event that is fired when an EntityItem's age has reached its maximum
- * lifespan. Canceling this event will prevent the EntityItem from being
- * flagged as dead, thus staying it's removal from the world. If canceled
- * it will add more time to the entities life equal to extraLife.
+ * lifespan. Adding extra life time will prevent the EntityItem from being
```

Comment:
Since you're touching the docs anyway: please replace `EntityItem` with `{@link ItemEntity}` throughout the class

Reply:
Will do...once my installation has recovered from switching branches.

Reply:
Done